js/cli/src/commands/options.ts

```diff
@@ -213,10 +213,15 @@ export const parseRunParametersArgument = (args: string[]): Record<string, strin
   return parsedParameters;
 };
 
-export const urlOption = new Option("--url <value>", "URL of Gatling Enterprise")
+export const apiUrlOption = new Option("--apiUrl <value>", "URL of the Gatling Enterprise API")
+  .default("https://api.gatling.io")
+  .hideHelp();
```

Comment:
Not backwards-compatible (changes the CLI option), but this setting was never documented for end users (`.hideHelp()`).